mlflow/store/tracking/abstract_store.py

```diff
@@ -239,28 +239,50 @@ def restore_run(self, run_id):
         """
         pass
 
-    def create_trace_info(
+    def start_trace(
         self,
         experiment_id: str,
         timestamp_ms: int,
-        execution_time_ms: int,
-        status: TraceStatus,
         request_metadata: Dict[str, str],
         tags: Dict[str, str],
     ) -> TraceInfo:
         """
-        Create a TraceInfo object under the specified experiment ID.
+        Start an initial TraceInfo object in the backend store.
 
         Args:
             experiment_id: String id of the experiment for this run.
             timestamp_ms: int, start time of the trace, in milliseconds.
-            execution_time_ms: int, duration of the trace, in milliseconds.
-            status: `mlflow.entities.TraceStatus`, status of the trace.
-            request_metadata: metadata of the trace.
-            tags: tags of the trace.
+            request_metadata: dict, metadata of the trace.
+            tags: dict, tags of the trace.
+
+        Returns:
+            The created TraceInfo object.
+        """
+        raise NotImplementedError
+
+    def end_trace(
+        self,
+        request_id: str,
+        timestamp_ms: int,
+        status: TraceStatus,
+        request_metadata: Dict[str, str],
+        tags: Dict[str, str],
+    ) -> TraceInfo:
+        """
+        Update the TraceInfo object in the backend store with the completed trace info.
+
+        Args:
+            request_id: Unique string identifier of the trace.
+            timestamp_ms: int, end time of the trace, in milliseconds. The execution time field
+                in the TraceInfo will be calculated by subtracting the start time from this.
+            status: TraceStatus, status of the trace.
+            request_metadata: dict, metadata of the trace. This will be merged with the existing
+                metadata logged during the start_trace call.
+            tags: dict, tags of the trace. This will be merged with the existing tags logged
+                during the start_trace or set_trace_tag calls.
```

Comment:
```suggestion
            request_id (str): Unique string identifier of the trace.
            timestamp_ms (int): end time of the trace, in milliseconds. The execution time field
                in the TraceInfo will be calculated by subtracting the start time from this.
            status (TraceStatus): status of the trace.
            request_metadata (dict): metadata of the trace. This will be merged with the existing
                metadata logged during the start_trace call.
            tags (dict): tags of the trace. This will be merged with the existing tags logged
                during the start_trace or set_trace_tag calls.
```

nit: to follow the google docstring style.

Reply:
I see, the original docstring already looks like this.

Reply:
Oh interesting, actually what about just removing those type note here? It's already obvious in type hint and it seems other APIs also do that.